docs/guide/authjs/session-data.md

```diff
@@ -0,0 +1,118 @@
+# Session data
+
+This guide explains how to add custom data to the user session. In many cases, you may wish to adapt which information is returned by the authention flow. This can depend on your provider or any additional API calls you may make, to enricht the session. 
```

Comment:
```suggestion
This guide explains how to add custom data to the user session. In many cases, you may wish to adapt which information is returned by the authentication flow. This can depend on your provider or any additional API calls you may make to enrich the session data.
```